ports/atmel-samd/boards/circuitplayground_express_crickit/mpconfigboard.h

```diff
@@ -32,6 +32,8 @@
 // Increase stack size slightly due to CPX library import nesting
 #define CIRCUITPY_DEFAULT_STACK_SIZE  (4760) // divisible by 8
 
+#define MICROPY_PY_REVERSE_SPECIAL_METHODS (0)
```

Comment:
I think this could be in circuitpy_mpconfig.h, forced on if ULAB is on.

Reply:
Ok, I moved it back to mpconfigport.h where the set to 0 was originally for SAMD21. The global defaults stay as they were, FULL_BUILD with a warning of ULAB on without it.